src/recipe/recipe.controller.ts

```diff
@@ -58,37 +58,29 @@ export class RecipeController {
     return await this.recipeService.updateRecipe(Number(recipeId), newRecipe);
   }
 
-  @Get('/id/:id')
-  @ApiOperation({ summary: 'Get one recipe by id' })
-  @ApiResponse({ type: RecipeResponse })
-  @ApiParam({ name: 'id', required: true })
-  async getOneRecipe(@Param('id') recipeId: number): Promise<RecipeResponse> {
-    return await this.recipeService.findRecipeById(Number(recipeId));
-  }
-
-  @Get('/list')
-  @UseGuards(AuthGuard('jwt'), IsAdminGuard)
+  @Get()
+  @UseGuards(AuthGuard('jwt'))
   @ApiOperation({ summary: 'Get all recipes' })
   @ApiResponse({ type: [RecipeResponse] })
   @ApiUnauthorizedResponse({ description: 'Not logged in' })
   @ApiForbiddenResponse({
     description: 'User is not an admin',
   })
   async getAllRecipes(
-    @Query('page') page: string,
-    @Query('limit') limit: string,
+    @Query() query: QueryParameters,
   ): Promise<RecipeResponse[]> {
-    return await this.recipeService.findAllRecipes(Number(limit), Number(page));
+    return await this.recipeService.findAllRecipes(
```

Comment:
```suggestion
    return this.recipeService.findAllRecipes(
```

Reply:
in general, a good idea would be to change `findAllRecipes` params type to take the whole query object. Than you can restructure it inside the service